Sources/ArcGISToolkit/Components/OfflineMapAreasView/OfflineMapAreasView.swift

```diff
@@ -56,6 +54,7 @@ public struct OfflineMapAreasView: View {
             }
             .task {
                 await mapViewModel.makePreplannedOfflineMapModels()
+                await mapViewModel.checkCanShowNotifications()
```

Comment:
These two function calls should be in separate task blocks. Otherwise the second won't happen until the first is done and these functions can execute in parallel.